baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -1033,4 +1037,4 @@ export class AssetServiceClient {
     }
     return Promise.resolve();
   }
-}
+}
```

Comment:
Looks like the empty line in the end of the template disappeared. Can you please add it back and rerun `npm run baseline`?